iris-mpc-cpu/tests/e2e.rs

```diff
@@ -0,0 +1,96 @@
+use eyre::Result;
+use iris_mpc_common::test::{load_test_db, TestCaseGenerator};
+use iris_mpc_cpu::execution::hawk_main::{HawkActor, HawkArgs, HawkHandle};
+use std::time::Duration;
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+const DB_SIZE: usize = 8 * 1000;
+const DB_RNG_SEED: u64 = 0xdeadbeef;
+const INTERNAL_RNG_SEED: u64 = 0xdeadbeef;
+const NUM_BATCHES: usize = 30;
+const MAX_BATCH_SIZE: usize = 64;
+const HAWK_REQUEST_PARALLELISM: usize = 1;
+const MAX_DELETIONS_PER_BATCH: usize = 10;
+
+fn install_tracing() {
+    tracing_subscriber::registry()
+        .with(
+            tracing_subscriber::EnvFilter::try_from_default_env().unwrap_or_else(|_| "info".into()),
+        )
+        .with(tracing_subscriber::fmt::layer())
+        .init();
+}
+
+async fn start_hawk_node(args: &HawkArgs) -> Result<HawkHandle> {
+    tracing::info!("🦅 Starting Hawk node {}", args.party_index);
+    let mut hawk_actor = HawkActor::from_cli(args).await?;
+    {
+        let (mut iris_loader, _graph_loader) = hawk_actor.as_iris_loader().await;
+        load_test_db(args.party_index, DB_SIZE, DB_RNG_SEED, &mut iris_loader)?;
+        // TODO: built the graph for the test db...
```

Comment:
@naure would be great to get some help on how to do this here

Reply:
Just to be clear we are doing that in a follow-up PR, does that work?

Reply:
yes, I will then merge this one.